README.md

```diff
@@ -7,12 +7,31 @@ The idea behind `kube-vip` is a small self-contained Highly-Available option for
 - Edge (arm / Raspberry PI)
 - Pretty much anywhere else :)
 
-*Use at your own risk, although it's pretty stable :-) *
-
 ![](/overview.png)
 
 The `kube-vip` application builds a multi-node cluster using [raft](https://en.wikipedia.org/wiki/Raft_(computer_science)) clustering technology to provide High-Availability. When a leader is elected, this node will inherit the Virtual IP and become the leader of the load-balancing within the cluster. 
 
+## Why?
+
+The purpose of `kube-vip` is to simplify the building of HA Kubernetes clusters, which at this time can involve a few components and configurations that all need to be managed. This was blogged about in detail by [thebsdbox](https://twitter.com/thebsdbox/) here -> [https://thebsdbox.co.uk/2020/01/02/Designing-Building-HA-bare-metal-Kubernetes-cluster/#Networking-load-balancing](https://thebsdbox.co.uk/2020/01/02/Designing-Building-HA-bare-metal-Kubernetes-cluster/#Networking-load-balancing).
+
+### Alternative HA Options
+
+`kibe-vip` provides both a floating or virtual IP address for your kubernetes cluster as well as load-balancing the incoming traffic to various control-plane replicas. At the current time to replicate this functionality a minimum of two pieces of tooling would be required:
+
+**VIP**:
+- [Keepalived](https://www.keepalived.org/)
+- [Ucarp](https://ucarp.wordpress.com/)
+- Hardware Load-balancer (functionality differs per vendor)
+
+
+**LoadBalancing**:
+- [HAProxy](http://www.haproxy.org/)
+- [Nginx](http://nginx.com)
+- Hardware Load-balancer(functionality differs per vendor)
+
+All of these would require a separate level of configuration and in some infrastructures multiple teams in order to implement. Also when considering the software components, they may require packaging into containers or if they’re pre-packaged then security and transparency may be an issue. Finally, in edge environments we may have limited room for hardware (no HW load-balancer) or packages solutions in the correct architectures might not exist (e.g. ARM). Luckily with `kibe-vip` being written in GO, it’s small(ish) and easy to build for multiple architectures, with the added security benefit of being the only thing needed in the container.
```

Comment:
again, reference to kibe-vip, I assume should be kube-vip?

Reply:
fixed